holoviews/plotting/mpl/__init__.py

```diff
@@ -221,18 +221,20 @@ def grid_selector(grid):
 
 options = Store.options(backend='matplotlib')
 dflt_cmap = 'fire'
+dflt_chart_cmap = 'kbc_r'
+
 # Default option definitions
 # Note: *No*short aliases here! e.g use 'facecolor' instead of 'fc'
 
 # Charts
 options.Curve = Options('style', color=Cycle(), linewidth=2)
-options.Scatter = Options('style', color=Cycle(), marker='o', cmap=dflt_cmap)
+options.Scatter = Options('style', color=Cycle(), marker='o', cmap=dflt_chart_cmap)
```

Comment:
What about `Path`?

Reply:
@jbednar apparently Path already has viridis as a default colormap. Maybe it should be consistent?

Reply:
Path should be consistent.